docs/api.yml

```diff
@@ -0,0 +1,136 @@
+openapi: 3.0.0
+
+info:
+  title: OSM Teams
+  description: Teams for OpenStreetMap!
+  version: 0.0.0
+
+servers:
+  - url: https://dev.mapping.team/api
+    description: Beta version of the API
+
+consumes:
+  - application/json
+
+produces:
+  - application/json
+
+tags:
+  - name: teams
+    description: Teams
+  - name: clients
+    description: Clients
+
+components:
+  schemas:
+    Team:
+      properties:
+        id:
+          type: integer
+          format: int64
+        name:
+          type: string
+        hashtag:
+          type: string
+        bio:
+          type: string
+        privacy:
+          type: string
+          enum: [private, public]
+        require_join_request:
+          type: boolean
+        created_at:
+          type: string
+          format: date-time
+        updated_at:
+          type: string
+          format: date-time
+        location:
+          type: string
+          description: geojson point
+    Client:
```

Comment:
I'd like to make this more detailed. There might be similar docs somewhere in the hydra project?

Reply:
was able to update this schema based on https://github.com/ory/hydra/blob/master/docs/api.swagger.json#L2365